app/main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
```

Comment:
skills variable should be an attribute of the instance not class. Follow checklist